dbTest.js

```diff
@@ -0,0 +1,35 @@
+const db = require('./models')
+
+
+// db.comment.create({
+//     name: 'Paul Allen',
+//     content: 'This is really neat! Thanks for posting.',
+//     articleId: 1
+//   })
+//   .then(function(comment) {
+//     console.log(comment.get())
+//   })
+
+// db.article.findOne({
+//     where: { id: 1 },
+//     include: [db.comment]
+//   }).then(function(article) {
+//     // by using eager loading, the article model should have a comments key
+//     console.log(article.comments)
+//   })
+
+
+
+  db.article.findAll({
+    include: [db.comment],
+    include: [db.article]
```

Comment:
Did this work for you, having two includes?  If you look in your articles.js file, you have the single include with both db.comment and db.article inside of it.  I've only seen it done that way, but haven't tried to use include twice like here in your dbTest.js file.

Reply:
Oh, I didn't even realize I had it like that until now but when I was doing the homework, everything worked fine.